src/components/CbApplet/CbAppletTarget.vue

```diff
@@ -9,22 +9,22 @@
       :page="page"
       :area="area"
       :context="context"
-      :user="appletUser"
+      :user="user"
       v-bind="$attrs"
     />
   </VExpandTransition>
 </template>
 
 <script setup>
-import { setActivePinia, storeToRefs } from 'pinia'
+import { setActivePinia } from 'pinia'
 import { computed, onBeforeMount, toRefs, watch } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { useAppletsStore } from '../../stores/applets'
 import CbApplet from './CbApplet.vue'
 
 const props = defineProps({
   /** String identifier for a specific applet id to target */
-  id: {
+  appletId: {
```

Comment:
So you don't mean this to be the `position`, but the actual `id` of the actual applet? Like, when the HUI uses this, the HUI will have hard-coded `<CbAppletTarget page="pageA" area="areaB" id="APL-123456" />`? I thought this was where you passed in the applet position within the area, like `pre-overview-tab`.

That feels weird to me: imho the source code for an application shouldn't get hard-coded with values for dynamic data like applet IDs. What do you see as the use-case for that?

Reply:
Alyssa sent me a video in slack explaining. Boiling it down, the use-case is for server-side rendering (like Django) to do the work of selecting a specific applet to render in a specific target, then using that rendered html to pass in the applet id. Works for me!